routes/pages/gate_routes.js

```diff
@@ -9,7 +9,8 @@ const Event = require('../../models/event').Event;
 router.get('/', userRole.isGateManager(), function (req, res) {
     Event.forge({event_id: req.user.currentEventId}).fetch().then(event => {
         return res.render('pages/gate', {
-            gate_code: event.attributes.gate_code
+            gate_code: event.attributes.gate_code,
```

Comment:
Why keep the `gate_code` here?

Reply:
no special reason, just wasn't sure whether or not it used elsewhere.

Reply:
Ok! looks good IMO - I updated the branch and if it passes ill merge 👍